beetmoverscript/tests/test_task.py

```diff
@@ -150,6 +152,49 @@ def test_validate_bucket_paths(bucket, path, raises):
         validate_bucket_paths(bucket, path)
 
 
+@pytest.mark.parametrize(
+    "payload_version,filename_version,folder_version,expectation",
+    (
+        (None, None, None, does_not_raise()),
+        ("a.bad.version", None, None, pytest.raises(ScriptWorkerTaskException)),
+        (None, "a.bad.version", None, pytest.raises(ScriptWorkerTaskException)),
+        (None, None, "a.bad.version", pytest.raises(ScriptWorkerTaskException)),
+    ),
+)
+def test_check_maven_artifact_map(context, payload_version, filename_version, folder_version, expectation):
+    context.action = "push-to-maven"
+
+    fake_correct_version = "12.3.20200920201111"
+    source = "fake_path/fake-artifact-{version}.jar"
+
+    payload_version = payload_version or fake_correct_version
```

Comment:
Nit: Let's replace the `None`s in parametrize with `"12.3.20200920201111"`. It will make the test even smaller and it will be easier to understand the test case, because the next reader won't have to mentally map `None` to the real value when this person reads lines 158-161.

Reply:
OK. I was trying to make it obvious that those values have to be the same and avoiding repeating the string literal by using the None's as a replacement for using a variable, since I didn't want to create a top-level variable for this single test neither.